app/views/workandbenefits/ShareSchemesAgentView.scala.html

```diff
@@ -32,10 +38,21 @@
             @govukErrorSummary(ErrorSummaryViewModel(form))
         }
 
+        @heading(messages("shareSchemes.agent.heading"))
+
+        @p(messages("shareSchemes.agent.p1"))
+
+        @bullets(Seq(
+            Html(messages("shareSchemes.bullet1")),
+            Html(messages("shareSchemes.bullet2"))
+        ))
```

Comment:
Im not sure if we are adding this but the bullet points might need padding to the left?